src/applications/simple-forms/21-0966/containers/ConfirmationPage.jsx

```diff
@@ -23,6 +29,31 @@ export class ConfirmationPage extends React.Component {
     const submitDate = submission.timestamp;
     const confirmationNumber = submission.response?.confirmationNumber;
 
+    const title = 'You’ve submitted your intent to file request';
+    const claimType = getClaimType(data);
+    const expirationDate = new Date(
```

Comment:
create a ticket or fix: if they bookmark this page, and come back to this page another time, will this date be wrong? is there anything that could be used in the submission data instead?